README.md

```diff
@@ -1,2 +1,47 @@
 # documentation-hydrus
-Sphinx documentation, for deploying in readthedocs.org
+This reposity contains documentation of [hydrus](https://github.com/HTTP-APIs/hydrus) built using [Sphinx](http://www.sphinx-doc.org/en/master/#)
+
+### Steps to build doc
+
+* Clone this repository.
+```
+$ git clone https://github.com/HTTP-APIs/documentation-hydrus.git
+```
+
+* Clone the [hydrus](https://github.com/HTTP-APIs/hydrus) repository.
+```
+$ git clone https://github.com/HTTP-APIs/hydrus.git
+```
+
+* It is recommended to use [virtualenv](https://virtualenv.pypa.io/en/stable/installation/) and [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/install.html) to maintain a clean environment.
+```
+$ source `which virtualenvwrapper.sh`
+$ mkvirtualenv -p python3 hydrus
+(hydrus) $ deactivate         # To deactivate the virtual environment
+$ workon hydrus               # To activate virtual environment
+```
+
+* Install requirements
+```
+(hydrus)$ cd documentation-hydrus 
+(hydrus)$ pip install -r requirements.txt
+```
+
+* Install hydrus
+```
+(hydrus)$ cd hydrus           # Go to your hydrus folder
+(hydrus)$ pip install -e .    # To install hydrus
+```
+
+* Copy the doc folder from documentation-hydrus folder to hydrus folder (Important step).
+
+* Everything is done now you can build the doc's .
+```
+(hydrus)$ cd hydrus/doc                 # The doc folder which was copied from documentation-hydrus to hydrus
+(hydrus)$ make html
```

Comment:
It is not working fine at all.
for fine working or get the page same as http://hydrus.readthedocs.io/en/latest/
you can go to rst folder(not necessary) but run the command `sphinx-build -b html path/to/hydrus/doc/rst  path/to/hydrus/doc/_build/html`(necessary to get rst files)
because all rst files is in folder rst which is required and necessary for documentation.

Reply:
ok adding it thanks!